docs/resources/workflow_task_archive_slack_channels.md

```diff
@@ -29,10 +29,9 @@ resource "rootly_workflow_task_archive_slack_channels" "archive_slack_channels"
   skip_on_failure = false
   enabled         = true
   task_params {
-    name = "Archive Slack channels"
     channels {
```

Comment:
@yjperez 
`name` here is referring to the task name its still required

`channels.name` should be `{{ incident.slack_channel_name }}`

Reply:
https://github.com/rootlyhq/terraform-provider-rootly/pull/38#issuecomment-1604600022